downstream-changes/update.sh

```diff
@@ -0,0 +1,81 @@
+#/bin/bash
+set -eux -o pipefail
+
+UPSTREAM_CLONE_URL=${UPSTREAM_CLONE_URL:-https://github.com/istio/istio.git}
+DOWNSTREAM_CLONE_URL=${DOWNSTREAM_CLONE_URL:-https://github.com/openshift-service-mesh/istio.git}
+REPO=$(basename "${UPSTREAM_CLONE_URL}" .git)
+BRANCHES=${BRANCHES:-master release-1.23 release-1.24}
```

Comment:
I guess we can remove release-1.23 as it's not used downstream

Reply:
done!